### ISSUE TITLE
Fight Caves Plugin is not functional

### ISSUE THREAD

**Issue by src52** [runelite/runelite#603]
I made it all the way to Jad, and I thought a Plugin would appear, but nothing ever showed up.

**Comment by Kamielvf** [comment]
I have failed to replicate this issue. Tried on 1.2.15, 1.2.16 and 1.2.17 and they all had the plugin working correctly.

![image](https://user-images.githubusercontent.com/35824069/36451252-b90c0612-1690-11e8-9c70-dfc0c7c71c49.png)

**Comment by tylerthardy** [comment]
@Kamielvf Did you try with all 3 possibly layout configurations? (Fixed, resizable with buttons on bottom, and resizable with buttons around inventory)

**Comment by Kamielvf** [comment]
@tylerthardy I only tested fixed and resizable with what you see in the screenshot

**Comment by deathbeam** [comment]
I dont think it matters, because the viewport is still same, only difference there is between positioning the overlay is between resizeable-* and fixed

**Comment by devinfrench** [comment]
The overlay only appears if you have the wrong prayer activated. Perhaps this can be changed so there is a constant overlay during the fight.

**Comment by deathbeam** [comment]
Ooooh. That would explain it.

**Comment by maaaaaaaaage** [comment]
This feature still active? , can’t see it on the list of features

**Comment by MagicfTail** [comment]
It very much should be. Our wiki/feature page doesn't have all features at the moment

**Comment by maaaaaaaaage** [comment]
Ah that’s where I looked, thanks guys

**Comment by TheClapTrap** [comment]
last night when fighting Jad the overlay never showed even when I had the wrong prayer on, help?

**Comment by Kamielvf** [comment]
make sure you have the plugin turned on. About 99% of the time when people reported it not working was when they either had it turned off, or Jad wasn't in range when they got hit by an attack (the plugin needs to see jad attack to know what atk style it uses)

**Comment by mike-colgan** [comment]
Just had this same issue today, stopped before jad to make sure it was on. Nothing appeared when jad spawned, even when in range.

**Comment by Nightfirecat** [comment]
I didn't see any issues on my Jad task today. Were you also using fixed mode, or resizable?

![2018-05-26_12-58-00](https://user-images.githubusercontent.com/2199511/40579795-85e6d21a-611f-11e8-89dd-d6ae725b14a6.png)

**Comment by mike-c13** [comment]
fixed mode.

**Comment by Mycrawsbow** [comment]
and they removed this feature because a bunch of idiots complained about it after they used it to get there cape they wanted to screw everyone else over i had it already how ever for slayer i really liked this plug in and the removed it. exactly why i dont donate money to any online sites the only features i really like get removed ha!
